packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -175,11 +183,36 @@ export interface ChatMessageProps
 
   /** Positions an actionMenu slot in "fixed" mode. */
   unstable_overflow?: boolean;
+
+  /** A message can render with different layouts. */
```

Comment:
Some slots & props work only for `layout="refresh"`. Can you please indicate this in their description?

Reply:
Sure. I'd originally prefixed all of those props (e.g. `v2_bubbleInset` or `refresh_bubbleInset`) to signal that, but it was suggested that I remove those. I'll go ahead and add that note to each of the props descriptions.